docs/src/pages/landing-page/QuasarComponents.vue

```diff
@@ -123,5 +161,23 @@ $number-of-card-columns-xs-max: 1;
   left: -4%;
   border-bottom: 1px solid $lp-primary;
   top: 40px;
+  z-index: 3;
+}
+
+.component-chip {
+  margin-left: 20px;
+}
+
+.large-screen-margin {
+  margin: 64px;
+}
+
+.chips-container {
+  @media screen and (min-width: $breakpoint-xs-max) {
```

Comment:
Chips should always be sticky, and the search bar should be too.
Nick acknowledged there will be the need to fix the large space they take into mobile viewports, but it's not a priority now